### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by steins-code**
update from translation

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 "ExposureDetection_Refreshed_Never" = "Unbekannt";
 
-"ExposureDetection_RefreshingIn" = "Aktualisierung in %02d:%02d\U00A0Minuten";
+"ExposureDetection_RefreshingIn" = "Aktualisierung in %02d:%02d\\U00A0Minuten";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:43]: erm, nope

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -521,7 +521,7 @@
 
 "Home_Risk_No_Date_Title" = "Unbekannt";
 
-"Home_Risk_Period_Update_Title" = "Aktualisierung alle %@\U00A0Stunden";
+"Home_Risk_Period_Update_Title" = "Aktualisierung alle %@\\U00A0Stunden";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: nope

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -39,7 +39,7 @@
 
 "ExposureDetection_Refreshed_Never" = "Unknown";
 
-"ExposureDetection_RefreshingIn" = "Update in %02d:%02d\U00A0minutes";
+"ExposureDetection_RefreshingIn" = "Update in %02d:%02d\\U00A0minutes";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings]: nope

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 4]
```diff
@@ -182,7 +182,7 @@
...
+"OnboardingInfo_enableLoggingOfContactsPage_panelBody" = "<Translation to be added>";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:185]: careful here

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 5]
```diff
@@ -514,7 +514,7 @@
 
 "Home_Risk_No_Date_Title" = "Unknown";
 
-"Home_Risk_Period_Update_Title" = "Updated every %@\U00A0hours";
+"Home_Risk_Period_Update_Title" = "Updated every %@\\U00A0hours";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings]: nope

**Review by eisvogel — COMMENTED** [review]
The new keys added yesterday are not yet included. Is this intended?

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: done

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 3 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings]: done

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 4 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:185]: This came from translation folks

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 5 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings]: done

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 4 above)
Comment (EisvogelSAP) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:185]: meaning I shall approve?

**Review by eisvogel — APPROVED** [review]

**Review by haosap — APPROVED** [review]

**Pull request merged**